themes/godotengine/pages/showcase/example-game.htm

```diff
@@ -0,0 +1,58 @@
+title = "Showcase | Example Game"
+url = "/showcase/example-game"
+layout = "showcase-item"
+description = "Example game showcase page"
+is_hidden = 0
+==
+{##}
+{% put title %} Example Game {% endput %}
+{% put author %} Jane Doe {% endput %}
+{% put author_url %} https://example.com/ {% endput %}
+{% put release_date %} October 2020 {% endput %}
+{% put platforms %} windows, macos, linux, android, ios, html5 {% endput %}
+
+{% put description %}
+  <p>
+    Example Game is a multiplayer party game where you have to find the other
+    suspicious players while not falling.
+  </p>
+  <p>
+    Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat eaque quia
+    quas maxime earum sed id qui! Magni quam voluptatum, voluptate ut laboriosam
+    officia voluptates delectus sed, nisi quibusdam soluta.
+  </p>
+  <h4>Key features</h4>
+  <ul>
+    <li>Over 25 carefully designed levels!</li>
+    <li>Create your own levels using the map editor!</li>
+    <li>Online multiplayer with up to 16 players!</li>
+  </ul>
+  <p>
+    Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat eaque quia
+    quas maxime earum sed id qui! Magni quam voluptatum, voluptate ut laboriosam
+    officia voluptates delectus sed, nisi quibusdam soluta. Lorem ipsum dolor
+    sit amet consectetur adipisicing elit.
+  </p>
+  <p>
+    Placeat eaque quia quas maxime earum sed id qui! Magni quam voluptatum,
+    voluptate ut laboriosam officia voluptates delectus sed, nisi quibusdam
+    soluta.
+  </p>
+{% endput %}
+
+{% put image %} https://cdn.cloudflare.steamstatic.com/steam/apps/404790/ss_f536b553fe0ec777c6ff89e968ac17b9d79341aa.jpg {% endput image %}
+{% put youtube_embed %} https://www.youtube-nocookie.com/embed/UEDEIksGEjQ {% endput %}
+{% put link %} https://store.steampowered.com/app/404790/ {% endput %}
```

Comment:
Probably makes sense to support multiple links? Not sure if `put`/`placeholder` would allow us that. Maybe we can bake in special `link_steam`, `link_gog`, `link_egs`, etc and allow one `link_custom` for something unsupported, like a personal page with the humble widget.